### DISCUSSION TITLE
Subscriptions and shortcuts on Yattee app for macOS

### DISCUSSION THREAD

**Discussion by ghost** [yattee/yattee#262] [General]
Hi,
I'm testing the Yattee app and I'm really enjoying it. However, I have some questions that I would be very grateful if answered.
1- To access subscriptions it is necessary to create an account. I created an account, but I would like to know where the subscriptions are stored. Are they stored on my computer or on a server? If they are stored on my computer, what is the need for the account?
2- I really like using shortcuts. For example, in Yattee, to perform a search, I need to use the shortcut "Cmd+F", but do not immediately select the "Search" field. For that I need to use the "Tab" key to select that field.
3- If I minimize Yattee and Yattee Player with the shortcut "Cmd+O" I can maximize the Yattee Player with "Cmd+O" , but I can't find any shortcut to get the Yattee app window.

Thank you and sorry for the inconvenience.

**Comment by arekf** [comment]
Hi,
1) Your subscriptions are stored by Invidious/Piped server. In the future I plan to implement local accounts.
2) That's a limitation of SwiftUI that I was unable to overcome in the past, I'll try to check soon, maybe somebody found a solution since then.
3) There's no such shortcut at the moment, I'll add it as an issue.

Cheers.

**Reply by ghost** [reply]
Hi @arekf,
Thank you for answering. If you don't mind,, I have another question.
When I delete Yattee and reinstall it accounts and instances appears again without doing nothing.
For example, using Finder, and enabling System Files on Finder search, if I search for Yattee I find and delete all files related to Yattee. However, If I install again Yattee the account and instances appears again. Can you please tell me where are stored accounts and instances?
Thank you.
Regards

**Reply by arekf** [reply]
Remove this folder:
 **/Users/your-username/Library/Containers/stream.yattee.app**
And run this command from terminal:
**defaults delete stream.yattee.app**

**Reply by ghost** [reply]
Hi again @arekf,
It works.Thank you.
Why inside /Users/your-username/Library/Containers/stream.yattee.app there is shortcuts to Desktop, Download, etc.?

If you allow my suggestion, try to integrate the Yattee Player on main Yattee window. For me a separate window for the player do not make any sense.

There are some bugs on Yattee preferences. For example, if I select the option to stop the video if I close Yattee player the video continues playing. To stop it I need to use "Cmd+P" or pause it with the mouse. There are others bugs on preferences, but I'm sure that they were already reported.

Btw, congratulations for the development of Yattee app.
Regards,

**Reply by arekf** [reply]
> Why inside /Users/your-username/Library/Containers/stream.yattee.app there is shortcuts to Desktop, Download, etc.?

Yattee is a sandboxed app on macOS meaning it cannot access anything on your hard drive except for this folder ("sandbox"). Those folders inside are created by macOS and access to them requires your approval (you must have seen before an alert like "App X is asking for access to your Documents/Downloads". These are in fact symlinks (shortcuts) to actual folders, so you can remove parent Yattee folder without fear that you'll lose contents of these folders linked inside.

**Reply by ghost** [reply]
> > Why inside /Users/your-username/Library/Containers/stream.yattee.app there is shortcuts to Desktop, Download, etc.?
> 
> 
> 
> Yattee is a sandboxed app on macOS meaning it cannot access anything on your hard drive except for this folder ("sandbox"). Those folders inside are created by macOS and access to them requires your approval (you must have seen before an alert like "App X is asking for access to your Documents/Downloads". These are in fact symlinks (shortcuts) to actual folders, so you can remove parent Yattee folder without fear that you'll lose contents of these folders linked inside.

Oh yes, I didn't remember that. Sorry.

**Reply by arekf** [reply]
For the preference for player in the same window and bugs, kindly open GitHub issues and I will be happy to fix. Thanks!

**Reply by ghost** [reply]
> For the preference for player in the same window and bugs, kindly open GitHub issues and I will be happy to fix. Thanks!

I will do that. Thank you.
